client/src/api/challenge.js

```diff
@@ -9,3 +9,13 @@ export async function countTasks() {
   const statistic = await response.json();
   return statistic;
 }
+
+export async function updateStatusTask(task) {
+  const response = await fetch(`/api/challenge/${task._id}`, {
+    method: "PATCH",
+    body: JSON.stringify(task.done),
+    headers: { "Content-Type": "application/json" },
```

Comment:
```suggestion
    headers: { "Content-Type": "application/json;charset=utf-8" },
```

Reply:
Thanks for this hint - I forgot it on every header 🙄